charts/drax/charts/e2-t/templates/deployment-e2-t-sctp.yaml

```diff
@@ -0,0 +1,33 @@
+{{- include
+      "accelleran.common.deployment"
+      (fromYaml (include "accelleran.e2-t.sctp.deployment.args" $))
+-}}
+
+{{- define "accelleran.e2-t.sctp.deployment.args" -}}
+{{- $ := . -}}
+{{- $values := index $.Values "e2-t-sctp" -}}
+
+top:
+  {{ $ | toYaml | nindent 2 }}
+values:
+  {{ $values | toYaml | nindent 2 }}
+
+initContainers:
+- {{ fromYaml (include "accelleran.common.init.redis" (fromYaml (include "accelleran.e2-t.init.args" .))) | toYaml | nindent 2 }}
+- {{ fromYaml (include "accelleran.common.init.nats" (fromYaml (include "accelleran.e2-t.init.args" .))) | toYaml | nindent 2 }}
+
+env:
+  - name: __APPNAME
+    value: sctpE2T
```

Comment:
We're inconsistent with the `__APPNAME` value for the e2t components - I don't know if it matters? (I'm not sure how the software actually uses these values)

Reply:
It doesn't really matter that much for the application (just makes monitoring nats connections more unique in combination with the `__APPID` in case only 0, 1, 2, ... is used there for example).

Reply:
This just made me think that we should include the release name (also for the CU and cell-wrapper)...

Reply:
In the CU-CP sctp is also used as a prefix in the __APPNAME, so I'll need to fix it there as well.